expenses/models.py

```diff
@@ -37,7 +37,7 @@ def create_expense(house_name, amount, date, category):
 
     @staticmethod
     def average_expenses_of_houses_by_categories(houses) -> QuerySet:
-        return (
-            Expenses.objects.filter(house_name__name__in=houses.values_list('name')).order_by().values(
-                'category').annotate(average=Avg("amount", output_field=IntegerField()))
-        )
+        average_expenses_of_houses_by_categories = Expenses.objects.filter(
```

Comment:
Please rename the variable name, it is confusing that both the method and the variable have the exact same name, also, not such a good convention